pom.xml

```diff
@@ -318,7 +318,7 @@
     <version.org.jboss.errai.wildfly>${version.org.wildfly}</version.org.jboss.errai.wildfly>
     <!-- Required by jboss-remoting version above -->
     <version.org.wildfly.security>1.10.4.Final</version.org.wildfly.security>
-    <version.org.ow2.asm>5.0.4</version.org.ow2.asm>
+    <version.org.ow2.asm>8.0.1</version.org.ow2.asm>
```

Comment:
we already are trying to upgrade to 7.1 at https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1233.

Reply:
I think 7.1 would probably suffice already too, so once #1233 is merged, I am happy to rebase and remove this line.

Reply:
ok